langfuse-node/src/langfuse-node.ts

```diff
@@ -25,9 +26,20 @@ export default class Langfuse extends LangfuseCore {
 
   private options: LangfuseOptions;
 
-  constructor(params: { publicKey: string; secretKey: string } & LangfuseOptions) {
-    const { publicKey, secretKey, ...options } = params;
-    super(params);
+  constructor(params?: { publicKey?: string; secretKey?: string } & LangfuseOptions) {
+    const { publicKey, secretKey, ...options } = utils.configLangfuseSDK(params);
+    if (!secretKey) {
+      throw new Error("secretKey is required");
```

Comment:
I assumed we do not want to throw here

Reply:
I kept it this way. Makes the PR much easier. This is basically the behavior from before. If we do not assess this at the start, we need to turn off the while SDK which i did not want to do in this PR. Otherwise we will flood our users with 401 logs.